internal/cmd/local/docker/docker.go

```diff
@@ -153,3 +175,205 @@ func (d *Docker) Port(ctx context.Context, container string) (int, error) {
 
 	return 0, errors.New("could not determine port for container")
 }
+
+const migratePGDATA = "/var/lib/postgresql/data"
+
+// MigrateComposeDB handles migrating the existing docker compose database into the abctl managed k8s cluster.
+// TODO: move this method out of the the docker class?
+func (d *Docker) MigrateComposeDB(ctx context.Context, volume string) error {
+	if v := d.volumeExists(ctx, volume); v == "" {
+		return errors.New(fmt.Sprintf("volume %s does not exist", volume))
+	}
+
+	// create a container for running the `docker cp` command
+	// docker run -d -v airbyte_db:/var/lib/postgresql/data alpine:3.20 tail -f /dev/null
+	conCopy, err := d.Client.ContainerCreate(
+		ctx,
+		&container.Config{
+			Image:      "alpine:3.20",
+			Entrypoint: []string{"tail", "-f", "/dev/null"},
+		},
+		&container.HostConfig{
+			Mounts: []mount.Mount{{
+				Type:   mount.TypeVolume,
+				Source: volume,
+				Target: migratePGDATA,
+			}},
+		},
+		nil,
+		nil,
+		"")
+	if err != nil {
+		return fmt.Errorf("could not create initial docker migrate container: %w", err)
+	}
+	pterm.Debug.Println(fmt.Sprintf("Created initial migration container '%s'", conCopy.ID))
+
+	// docker cp [conCopy.ID]]:/$migratePGDATA/. ~/.airbyte/abctl/data/airbyte-volume-db/pgdata
+	userHome, _ := os.UserHomeDir()
+	dst := filepath.Join(userHome, ".airbyte", "abctl", "data", "airbyte-volume-db", "pgdata")
```

Comment:
Suggestion: We could have a small package that is for managing and interacting with this the `$HOME/.airbyte` directory (e.g. check if folder exists, read file from path, etc). But it handles all of the Homedir resolution and filepath joining.

Reply:
I actually started the work on this in the #41 pr, for this very reason.  I'm planning to iterate on it further soon.